src/predict.py

```diff
@@ -13,20 +13,20 @@
 
 TEST_BATCH_SIZE = 64
 
+
 def prediction(batch_dispatcher, tf_vars, array_cost, pred_array, id_array):
```

Comment:
1. if it's a `list` we should maybe not call it `pred_array` and `id_array`. 
2. Will `pred_array` and `id_array` always be empty when this function is called? If so, I don't see a reason why they should be parameters.